docs/cloud/guides/production-readiness.md

```diff
@@ -0,0 +1,180 @@
+
+This guide walks through each area, helping you transition from a working ClickHouse Cloud deployment to an enterprise-ready system.
+
+## Environment Strategy {#environment-strategy}
+
+Establish separate environments to safely test changes before impacting production workloads. Most production incidents trace back to untested queries or configuration changes deployed directly to production systems.
+
+**Environment Structure**: Maintain production (live workloads), staging (production-equivalent validation), and development (individual/team experimentation) environments.
+
+**Testing**: Test queries in staging before production deployment. Queries that work on small datasets often cause memory exhaustion, excessive CPU usage, or slow execution at production scale. Validate configuration changes including user permissions, quotas, and service settings in staging—configuration errors discovered in production create immediate operational incidents.
+
+**Sizing**: Size your staging service to approximate production load characteristics. Testing on significantly smaller infrastructure may not reveal resource contention or scaling issues. Use production-representative datasets through periodic data refreshes or synthetic data generation.
+
+## Enterprise Authentication and User Management {#enterprise-authentication}
+
+Moving from console-based user management to enterprise authentication integration is essential for production readiness.
+
+### SSO/SAML Setup {#sso-saml-setup}
+
+Enterprise tier ClickHouse Cloud supports SAML integration with identity providers including Okta, Azure Active Directory, and Google Workspace. SAML configuration requires coordination with ClickHouse support and involves providing your IdP metadata and configuring attribute mappings.
+
+:::note Important limitation
+Users authenticated through SAML are assigned the "Member" role by default and must be manually granted additional roles by an admin after their first login. Group-to-role mapping and automatic role assignment are not currently supported.
+:::
+
+### Access Control Design {#access-control-design}
+
+ClickHouse Cloud uses organization-level roles (Admin, Developer, Billing, Member) and service/database-level roles (Service Admin, Read Only, SQL console roles). Design roles around job functions applying the principle of least privilege:
+
+- **Application Users**: Service accounts with specific database and table access
+- **Analyst Users**: Read-only access to curated datasets and reporting views
+- **Admin Users**: Full administrative capabilities
+
+Configure quotas, limits, and settings profiles to manage resource usage for different users and roles. Set memory and execution time limits to prevent individual queries from impacting system performance. Monitor resource usage through audit, session, and query logs to identify users or applications that frequently hit limits. Conduct regular access reviews using ClickHouse Cloud's audit capabilities.
```

Comment:
This tells me what to do but not how to do it. Can we cross link to docs on how to:
- configure quotas
- configure limits
- configure settings profiles
- set memory and execution time limits
- monitor resource usage